api/app/src/main/kotlin/packit/service/PacketGroupService.kt

```diff
@@ -28,15 +29,9 @@ class BasePacketGroupService(
         val packetGroups = packetGroupRepository.findAllByNameContaining(filteredName, Sort.by("name"))
         return PagingHelper.convertListToPage(packetGroups, pageablePayload)
     }
-
-    override fun getPacketGroupDisplay(name: String): PacketGroupDisplay{
-        val latestPacketId = packetGroupRepository.findLatestPacketIdForGroup(name)?.id
-        val packetGroup = packetGroupRepository.findByName(name)
-        if (latestPacketId == null || packetGroup == null) {
-            throw PackitException("doesNotExist", HttpStatus.NOT_FOUND)
-        }
-        val packetOrderlyMetadata = packetService.getMetadataBy(latestPacketId).custom?.get("orderly") as? Map<*, *>
-        val packetLongDescription = (packetOrderlyMetadata?.get("description") as? Map<*, *>)?.get("long") as? String
-        return PacketGroupDisplay(packetGroup.latestDisplayName, packetLongDescription)
+    override fun getPacketGroupDisplay(name: String): PacketGroupDisplayDto{
+        val packetGroupSummary = packetGroupRepository.getFilteredPacketGroupSummaries(name).firstOrNull()
+            ?: throw PackitException("No packets groups found for name $name", HttpStatus.NOT_FOUND)
```

Comment:
As discussed, I don't think this quite works, as `getFilteredPacketGroupSummaries` uses `ILIKE` so, if there was no exact match for `name`, it could return an inexact match instead, whereas we would want it to return a 404 in that case.